01-Login/src/auth/authWrapper.js

```diff
@@ -32,21 +32,23 @@ export const useAuth0 = ({
 
         try {
           await this.auth0Client.loginWithPopup(o);
+          this.user = await this.auth0Client.getUser();
+          this.isAuthenticated = await this.auth0Client.isAuthenticated();
```

Comment:
Is this something that might get updated while the app is still in memory? Wondering if this should be the `auth0Client.isAuthenticated` method itself rather than the current value ...

Reply:
Don't worry, it will be. `isAuthenticated` is a reactive Vue property, so if the state of authentication ever changes, we can just set this value and everything that is watching it will update (Vue components, anything using `$watch`, etc). We just make sure that its value is set whenever we check the authentication state inside the wrapper.